markdown/GameGuides/dishonored_death_of_the_outsider.md

```diff
@@ -0,0 +1,47 @@
+![DishonoredDeathoftheOutsider](\\DishonoredDeathoftheOutsider.png "Shot by Originalnicodr")
+
+
+## Summary
+
+Feature | Supported
+--|--
+Vanilla Photo Mode | No
+Hotsampling | No 
+DSR | No
+Custom Aspect Ratios | No
+Reshade | Yes 
+Ansel | Yes
+DirectX versions | DirectX 11
+ 
+## Tools
+
+* [Dishonored 2 / DotO camera tools by Frans Bouma](https://github.com/FransBouma/InjectableGenericCameraSystem/releases/tag/DH2_102)  
+**Features**: Free Camera, FOV, Time control, HUD toggle, DOF control, Bloom, Color and more
```

Comment:
These features are not correct: https://github.com/FransBouma/InjectableGenericCameraSystem/tree/master/Cameras/Dishonored2#features

Reply:
Ups, the problems that comes with copy-pasting.

Thanks for cheking the guides are correct Frans.